build/NuSpecs/MUXControls-Nuget-FrameworkPackage.props

```diff
@@ -3,68 +3,68 @@
   <Import Project="$(MSBuildThisFileDirectory)\MicrosoftUIXamlVersion.props"/>
   <Import Project="$(MSBuildThisFileDirectory)\Common.targets"/>
   <ItemGroup>
-    <AppxPackageRegistration Include="$(MSBuildThisFileDirectory)..\tools\AppX\x86\Release\Microsoft.UI.Xaml.2.1.appx">
+    <AppxPackageRegistration Include="$(MSBuildThisFileDirectory)..\tools\AppX\x86\Release\Microsoft.UI.Xaml.2.2.appx">
```

Comment:
Can we make 2.2 as a parameter? eg:
$(MSBuildThisFileDirectory)..\tools\AppX\x86\Release\$(MUXVersion).appx

Reply:
Yes, but I don't think it is worth doing because it just moves the peas around on the plate. The real problem is that I have to touch any file beyond just custom.props to bump the version. And I agree we should do that but at the moment the ROI is low since we only do this 3 times a year.